From 83a6742f08349fbc93f459228dcc3d1f56eac411 Mon Sep 17 00:00:00 2001
From: ycitgez <yuhanun.citgez@flatexdegiro.com>
Date: Tue, 12 Jul 2022 13:20:32 -0700
Subject: [PATCH] Resolved a compiler error due to lacking include (#162)

Summary:
Fixed an issue where a missing include causing compiler errors

Also applied clang-format.sh

Issue occured with following compiler and linker:
```
C++ compiler for the host machine: ccache c++ (gcc 12.1.1 "c++ (GCC) 12.1.1 20220507 (Red Hat 12.1.1-1)")
C++ linker for the host machine: c++ ld.bfd 2.37-27
```

Pull Request resolved: https://github.com/facebookincubator/oomd/pull/162

Reviewed By: brianc118

Differential Revision: D37790605

Pulled By: lnyng

fbshipit-source-id: d42776978b4bc8f7e2f584fde109e6cc3f5bc7d6
---
 src/oomd/Log.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/oomd/Log.h b/src/oomd/Log.h
index 3d2d6ea..0ed5f73 100644
--- a/src/oomd/Log.h
+++ b/src/oomd/Log.h
@@ -18,6 +18,7 @@
 #pragma once
 
 #include <algorithm>
+#include <array>
 #include <condition_variable>
 #include <iostream>
 #include <mutex>
